tests/metrics_publishing.rs

```diff
@@ -702,3 +675,16 @@ where
         Ok(())
     }
 }
+
```

Comment:
I don’t understand the issue yet, but the test failed because the order of attributes in the exported metrics differed each time. Therefore, I added a sorting process during the assertion.